source/php/Entity/PostTypeInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace VolunteerManager\Entity;
+
+interface PostTypeInterface
+{
+    public function addHooks(): void;
+
+    public function registerPostType(): void;
+
+    public function addTableColumn(string $key, string $title, bool $sortable = false, callable|false $contentCallback = false): bool;
```

Comment:
Union types are only allowed since PHP 8.0, according to phpStorm.
Should PHP 8 be allowed for the project?

Reply:
Fixed it now. Also changed return type to void because always return true makes no sense.